ui/app/[locale]/UserDropdown.tsx

```diff
@@ -27,16 +34,24 @@ export function UserDropdown({ username }: { username: string }) {
             isExpanded={isOpen}
             icon={
               <Avatar
-                src={"https://www.patternfly.org/images/668560cd.svg"}
-                alt=""
+                src={
+                  picture || "https://www.patternfly.org/images/668560cd.svg"
+                }
+                alt={username || "User"}
               />
             }
           >
-            {username}
+            {username || "User"}
           </MenuToggle>
         )}
       >
         <DropdownList>
+          <DropdownItem>
+            <a href={process.env.NEXT_PUBLIC_KEYCLOAK_URL + "/account/"}>
```

Comment:
Can this just use `KEYCLOAK_ISSUER` ? I think `${KEYCLOAK_ISSUER} + "/account/"` is the correct path anyway.